examples/twittermap/web/public/javascripts/common/services-postgresql.js

```diff
@@ -0,0 +1,289 @@
+angular.module('cloudberry.common', [])
```

Comment:
one more comment: this file and other `service-xx.js`s have the common query generator functions. could you write a `queryGenerator` module that takes care of the common generation logic? then the `service` model could be possibly reused. Thanks!